### PR TITLE
more harmony

### PR DESCRIPTION
**Pull request by schollz**

### CODE REVIEW

File: lib/parts.lua [hunk 1]
```diff
@@ -63,6 +63,12 @@ function parts.init(soundEngine)
     ['is_nervous'] = false
   }
 
+  -- chords
```
Comment (ryanlaws) [inline lib/parts.lua:66]: You can use `SoundEngine:getSetParam(id, name)` (to get a single-param setter) or `SoundEngine:setSynthParams(id, synthParams)` (to set more than 1 param at a time, where `synthParams` is a dict-table) if you like, too.

**Comment by ryanlaws** [comment]
Looks good! @schollz I made one comment about setting the params, but it's really just a suggestion. Things are in flux so you may not have seen those.

**Review by ryanlaws — CHANGES_REQUESTED** [review]
Oops, looks like we've got some conflicts - can you fix those?

File: lib/parts.lua (hunk 1 above)
Comment (schollz) [reply lib/parts.lua:66]: done!

**Comment by schollz** [comment]
force pushed! fixed now i think?

**Review by ryanlaws — APPROVED** [review]

**Comment by ryanlaws** [comment]
Groovy! 👍🏻

**Pull request merged**